### DISCUSSION TITLE
Error when creating dzi pyramids from tif file

### DISCUSSION THREAD

**Discussion by MstfTkrsln** [libvips/libvips#2049] [Q&A]
Trying to create dzi pyramids with following command, i am getting an error which is `"tiff2vips: required field 262 missing"`

What is the meaning of this error?

**Command**
`dzsave "C:\my_path\my_file.tif" "C:\my_dest_path\" --suffix .png --tile-size 1024`

**tif file properties**
![image](https://user-images.githubusercontent.com/21102136/91711527-b56e2780-eb8e-11ea-9454-a44433d51257.png)

**Comment by jcupitt** [comment]
Looks like it's this:

```
tiff.h:#define	TIFFTAG_PHOTOMETRIC		262	/* photometric interpretation */
```

That's a required field, so your TIFF file seems to be non-standard.

If the output looks OK, then libvips has probably guessed a reasonable default.

**Comment by jcupitt** [comment]
Wait, I guess it's refusing to read the file?

Sorry, there's not much that can be done. We'd need to add a lot of extra code to try to guess the setting for `PHOTOMETRIC_INTERPRETATION` from the other tiff fields. 

You could maybe use `tiffcp` to try to repair the file, but it'll probably refuse to read it as well.
